mux/bindings/python/cmux_mux_client/client.py

```
@@ -0,0 +1,541 @@
+        except json.JSONDecodeError as exc:
+            raise ProtocolError(f"bad JSON from server: {exc}") from exc
+
+    def close(self) -> None:
+        try:
+            self.reader.close()
+        finally:
+            self.sock.close()
+
+
+class _Stream:
+    def __init__(self, client: "MuxClient", request: Dict[str, Any]):
```

Ruff (0.15.20)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)